rest-api/dao/participant_summary_dao.py

```diff
@@ -282,6 +284,7 @@ def to_client_json(self, model):
       result = {k: result.get(k) for k in WITHDRAWN_PARTICIPANT_FIELDS}
 
     result['participantId'] = to_client_participant_id(model.participantId)
+    result['lastModified'] = model.lastModified
```

Comment:
I think this already happens with model.asdict() above? Is this not included already?

Reply:
correct, removed.